### PR TITLE
CNV-36794: Update vm example template

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Updating fedora version from 36 to latest

### CODE REVIEW

File: src/templates/vm-yaml.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ spec:
       volumes:
         - name: rootdisk
           containerDisk:
-            image: 'quay.io/containerdisks/fedora:36'
+            image: 'quay.io/containerdisks/fedora'
```
Comment (pcbailey) [inline src/templates/vm-yaml.ts:63]: Did you mean to add the latest tag?

Comment (metalice) [reply src/templates/vm-yaml.ts:63]: yes, changing to latest

**Pull request merged**